ros2topic/ros2topic/verb/info.py

```diff
@@ -29,7 +32,16 @@ def add_arguments(self, parser, cli_name):
 
     def main(self, *, args):
         with DirectNode(args) as node:
+            topic_names_and_types = get_topic_names_and_types(node=node)
             topic_name = args.topic_name
-            print('Topic: %s' % topic_name)
+            for (t_name, t_types) in topic_names_and_types:
+                if t_name == topic_name:
+                    topic_types = t_types
+                    break
+            else:
+                print("ERROR: Unknown topic '%s'" % topic_name, file=sys.stderr)
+                return
+            type_str = topic_types[0] if len(topic_types) == 1 else topic_types
+            print('Type: %s' % type_str)
             print('Publisher count: %d' % node.count_publishers(topic_name))
             print('Subscriber count: %d' % node.count_subscribers(topic_name))
```

Comment:
Unrelated, but we could improve the API, in order to get the publisher/subscription count for each specific type.